pkg/service/kubeconfig_settings_test.go

```diff
@@ -0,0 +1,164 @@
+	}{
+		{"Invalid OrgId", "Invalid OrgId", acuuid, true},
+		{"Invalid AccId", ouuid, "Invalid AccId", true},
+		{"Org id is empty", "", acuuid, true},
+		{"Acc id is empty", ouuid, "", true},
+	}
+
+	for _, tc := range tt {
+		t.Run(tc.name, func(t *testing.T) {
+			db, mock := getDB(t)
+			defer db.Close()
+
+			ps := NewKubeconfigSettingService(db)
+
+			uuuid := uuid.New().String()
+			validity_seconds := 300
+			disable_web_kubectl := true
+			disable_cli_kubectl := true
+
+			mock.ExpectQuery(`SELECT "ks"."id", "ks"."organization_id", "ks"."partner_id", "ks"."account_id", "ks"."scope", "ks"."validity_seconds", "ks"."created_at", "ks"."modified_at", "ks"."deleted_at", "ks"."enforce_rsid", "ks"."disable_all_audit", "ks"."disable_cmd_audit", "ks"."is_sso_user", "ks"."disable_web_kubectl", "ks"."disable_cli_kubectl", "ks"."enable_privaterelay", "ks"."enforce_orgadmin_secret_access" FROM "sentry_kubeconfig_setting" AS "ks" WHERE \(organization_id = '` + ouuid + `'\) AND \(account_id = '` + acuuid + `'\)`).
+				WithArgs().WillReturnRows(sqlmock.NewRows([]string{"id", "organization_id", "account_id", "validity_seconds", "disable_web_kubectl", "disable_cli_kubectl"}).AddRow(uuuid, ouuid, acuuid, validity_seconds, disable_web_kubectl, disable_cli_kubectl))
+
+			kss := &sentry.KubeconfigSetting{Id: uuuid, OrganizationID: ouuid, AccountID: acuuid, ValiditySeconds: int64(validity_seconds), DisableWebKubectl: disable_web_kubectl, DisableCLIKubectl: disable_cli_kubectl}
+
+			kss, err := ps.Get(context.Background(), tc.orgId, tc.accId, false)
+			if tc.shouldfail {
+				if err == nil {
+					t.Fatal("got kubeconfig setting for invalid ids")
+					t.Log(kss.GetId())
+				} else {
+					return
+				}
+			}
+		})
+	}
+}
+
+func TestGetKubeconfigSettingNoConfig(t *testing.T) {
+	db, mock := getDB(t)
+	defer db.Close()
+
+	ps := NewKubeconfigSettingService(db)
+
+	uuuid := uuid.New().String()
+	ouuid := uuid.New().String()
+	acuuid := uuid.UUID.String(uuid.New())
+	validity_seconds := 300
+	disable_web_kubectl := true
+	disable_cli_kubectl := true
+
+	mock.ExpectQuery(`SELECT "ks"."id", "ks"."organization_id", "ks"."partner_id", "ks"."account_id", "ks"."scope", "ks"."validity_seconds", "ks"."created_at", "ks"."modified_at", "ks"."deleted_at", "ks"."enforce_rsid", "ks"."disable_all_audit", "ks"."disable_cmd_audit", "ks"."is_sso_user", "ks"."disable_web_kubectl", "ks"."disable_cli_kubectl", "ks"."enable_privaterelay", "ks"."enforce_orgadmin_secret_access" FROM "sentry_kubeconfig_setting" AS "ks" WHERE \(organization_id = '` + ouuid + `'\) AND \(account_id = '` + acuuid + `'\)`).
+		WithArgs().WillReturnRows(sqlmock.NewRows([]string{"id", "organization_id", "account_id", "validity_seconds", "disable_web_kubectl", "disable_cli_kubectl"}))
+
+	kss := &sentry.KubeconfigSetting{Id: uuuid, OrganizationID: ouuid, AccountID: acuuid, ValiditySeconds: int64(validity_seconds), DisableWebKubectl: disable_web_kubectl, DisableCLIKubectl: disable_cli_kubectl}
```

Comment:
For a get request, we are fetching based on the ids, and won't have the settings pass on in the request.